packages/lodestar/src/cli/commands/interop.ts

```diff
@@ -116,23 +122,32 @@ export class InteropCommand implements ICliCommand {
     } else {
       peerId = createPeerId();
     }
+    const network: {multiaddrs?: string[]; bootnodes?: string[]} = conf.network || {};
+    const multiaddrs = network.multiaddrs || [];
+    const bootnodes = network.bootnodes || [];
     const libp2p = await Promise.resolve(peerId)
-      .then((peerId) => initializePeerInfo(peerId, conf.network.multiaddrs))
-      .then((peerInfo) => new NodejsNode({peerInfo, bootnodes: conf.network.bootnodes}));
+      .then((peerId) => initializePeerInfo(peerId, multiaddrs))
+      .then((peerInfo) => new NodejsNode({peerInfo, bootnodes: bootnodes}));
```

Comment:
@wemeetagain Why are we initializing libp2p here and not inside network module?

Reply:
We have these async functions we need to wait for. There was trouble awaiting them in the network constructor. The `NodejsNode` is just a libp2p instance.
Another thought was that we might want to reuse this instance for other beacon nodes or shard nodes.

Reply:
can we arange this so its awaited in start method of network?

Reply:
Before this solution, we created a promise in the BeaconNode constructor and passed in the promise to the network, which made sure it was finished before starting.

Reply:
Ill think about cleaner solution on monday

Reply:
Think about reusing the network instance between multiple nodes, too.

Reply:
I move it to a separate async factory function, please have a look.